src/compute-engine/library/arithmetic.ts

```diff
@@ -641,7 +641,6 @@ export const ARITHMETIC_LIBRARY: IdentifierDefinitions[] = [
           });
           return sumNeg % 2 === 0 ? 'non-positive' : 'non-negative';
         }
-        if (ops.every((x) => !x.is(0) && x.isReal)) return 'real-not-zero';
         if (ops.every((x) => !x.is(0))) return 'not-zero';
         if (ops.every((x) => x.isReal)) return 'real';
```

Comment:
There is a check for isReal at the beginning of the handler, so I don't think this is ever reached.

Reply:
I agree with your point.